README.md

```diff
@@ -13,5 +13,5 @@ This repository contains a templated `Dockerfile` for image variants designed to
 An example of how to use `cibuild` to build and test an image:
 
 ```bash
-$ CI=1 TERRAFORM_VERSION=9.8 ./scripts/cibuild
+$ CI=1 TERRAFORM_VERSION=0.10.8 ./scripts/cibuild
```

Comment:
Looks good, why `0.10.x` vs `0.11.x`?

Reply:
I saw that's the latest `0.10.x` release, https://hub.docker.com/r/hashicorp/terraform/tags/, assuming it's LTS?

Reply:
I just used `0.10.8`  because we're not building `0.11.x` yet.